configs/components/curl.rb

```diff
@@ -1,5 +1,5 @@
 component 'curl' do |pkg, settings, platform|
-  pkg.version '7.56.0'
+  pkg.version '7.56.1'
   pkg.md5sum '65351b9df687ed539852ae7b9464006c'
```

Comment:
Need to update the checksum. I suggest testing builds on a few platforms: a linux, windows, maybe solaris/aix.

Reply:
Gah, thanks @MikaelSmith, I'll update and push again once I've successfully built this on a few platforms.